src/components/ContributionsEpicReminder.tsx

```diff
@@ -125,66 +87,187 @@ const errorTextStyles = css`
     color: ${palette.error.main};
     font-style: italic;
     margin-top: ${space[1]}px;
+    margin-bottom: 0;
 `;
 
-export const ContributionsEpicReminder: React.FC<ReminderFields> = ({
+const getCustomSubmitStyles = (isDisabled: boolean): string | undefined => {
+    if (isDisabled) {
+        // Unfortunately these overrides need !important as otherwise they'll lose
+        // the specificity war against the default Source styles.
+        return css`
+            pointer-events: none;
+            color: ${palette.neutral[60]} !important;
+            background-color: ${palette.neutral[93]} !important;
+            border: 1px solid ${palette.neutral[86]} !important;
+        `;
+    }
+
+    return undefined;
+};
+
+const isValidEmail = (email: string): boolean => {
```

Comment:
Is there a centralised, common pattern for this anywhere?

Reply:
Ha, not sure. It is notoriously difficult to validate email addresses for sure.